app/models/payload.rb

```diff
@@ -29,8 +29,12 @@ def self.min_response
     minimum("responded_in")
   end
 
+  def parse_payload(params)
+    JSON.parse(params[:payload])
+  end
```

Comment:
@meganft This was the the parsed_payload (misspelled) that I was calling in the route. My thought was that since payload_constructor and already_exist? called the same thing to set input I would pull it out. Didn't pan out like i thought it would.